public/components/page-contributors/contributors/contributors.stache

```diff
@@ -10,24 +10,24 @@
     </tr>
   </thead>
   <tbody>
-  {{#each contributors as contributor}}
+  {{# each(contributors, contributor=value) }}
     <tr>
-      <td><model-edit-property model:from="contributor" property="name" /></td>
-      <td><model-edit-property model:from="contributor" property="email" /></td>
+      <td><model-edit-property model:from="contributor" property:from="'name'" /></td>
+      <td><model-edit-property model:from="contributor" property:From="'email'" /></td>
       <td>
-        <input type="checkbox" checked:from="active" on:change="setActive(contributor, %element.checked)"{{#if isSaving}} disabled{{/if}} />
+        <input type="checkbox" checked:from="active" on:change="../setActive(contributor, scope.element.checked)"{{# if(isSaving()) }} disabled{{/ if }} />
```

Comment:
There are multiple changes here:

- `setActive(...)` -> `../setActive(...)` -> The can-stache scope must be specified.  The current context is the contributor.  `setActive` is on the view model.  `scope.vm.setActive( ... )` could also have been used.